pylint_django/__init__.py

```diff
@@ -3,10 +3,17 @@
 
 import sys
 
-from pylint_django import plugin
+from pylint_django import plugin, compat
 
 if sys.version_info < (3, ):
     raise DeprecationWarning("Version 0.11.1 was the last to support Python 2. "
                              "Please migrate to Python 3!")
 
-register = plugin.register  # pylint: disable=invalid-name
+# Older versions of pylint does not support load_configuration() hook. To be
+# backward compatible we call load_configuration() directly from register()
+# hook. Using legacy approach yields issue #181.
+# For more details see issue #181 and PR #214.
+if compat.REGISTER_LEGACY:
```

Comment:
wait is this the right way round? `compat.REGISTER_LEGACY` sounds like it should call `plugin.register_legacy`. Might just be a naming issue

Reply:
Oh stupid mistake... I will fix it. In any case I would like to do some manual tests for this functionality + investigate whether we can print some warning when old pylint is used. Maybe some unittests can be added for this functionality what do you think?

Reply:
@carlio, @matusvalo - looking at the latest test results I see all PASS. See job 524.8 with env DJANGO=master. This is installing pylint from their master branch and the console log shows pylint version 2.3.0.

IMO we're OK on the testing side here.